cypress/e2e/swap.test.ts

```diff
@@ -154,6 +154,20 @@ describe('Swap', () => {
     cy.get('#swap-currency-output .token-amount-input').should('have.value', '0.01')
   })
 
+  it('should render and dismiss the wallet rejection modal', () => {
+    cy.stub(hardhat.wallet, 'sendTransaction').rejects(new Error('user cancelled'))
+    cy.get('#swap-currency-output .open-currency-select-button').click()
+    cy.get(getTestSelector('token-search-input')).clear().type(USDC_MAINNET.address)
+    cy.contains('USDC').click()
+    cy.get('#swap-currency-output .token-amount-input').clear().type('1')
+    cy.get('#swap-currency-input .token-amount-input').should('not.equal', '')
```

Comment:
we should know _exactly_ what it will equal, because we are on a fork

Reply:
I think this is a reasonable fudge, since in order to know exactly what we'd get here we would need to run the router and get the input amount for this exact output swap, and the routing api isn't looking at the fork